docs/source/training_tutorials/finetune_llm.mdx

```diff
@@ -0,0 +1,417 @@
+<!---
+Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Fine-tune and Test Llama-3 8B on AWS Trainium 
+
+_Note: The complete script for this tutorial can be downloaded [here](https://github.com/huggingface/optimum-neuron/docs/source/training_tutorials/finetune_llm.py)._
+
+This tutorial will teach you how to fine-tune open source LLMs like [Llama 3](https://huggingface.co/meta-llama/Meta-Llama-3-8B) on AWS Trainium. In our example, we are going to leverage the [Optimum Neuron](https://huggingface.co/docs/optimum-neuron/index), [Transformers](https://huggingface.co/docs/transformers/index) and [Datasets](https://huggingface.co/docs/datasets/index) libraries.
+
+You will learn how to:
+
+1. [Setup AWS Environment](#1-setup-aws-environment)
+2. [Load and process the dataset](#2-load-and-prepare-the-dataset)
+3. [Fine-tune Llama on AWS Trainium using the `NeuronTrainer`](#3-fine-tune-llama-on-aws-trainium-using-the-neurontrainer)
+4. [Launch Training](#4-launch-training)
+5. [Evaluate and test fine-tuned Llama model](#5-evaluate-and-test-fine-tuned-llama-model)
+
+
+<Tip>
+
+While we will use `Llama-3 8B` in this tutorial, it is completely possible to use other models, simply by swtiching the `model_id`.
+For instance, it is possible to fine-tune:
+
+- Mistral models, such as  [Mistral 7b (`mistralai/Mistral-7B-Instruct-v0.3`)](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2)
+- Llama-2 models, such as [Llama-2 7b (`meta-llama/Llama-2-7b-hf`)](https://huggingface.co/meta-llama/Llama-2-7b-hf)
+
+And many others!
```

Comment:
WDYT of adding a section in https://huggingface.co/docs/optimum-neuron/en/package_reference/supported_models for the ones we support for training? and reference it here

Reply:
Why not but it is a bit "unclear". Because:

1. We do not know which architecture is supported for regular training specifically.
2. The supported architectures for distributed training is known, we can add that.